### PR TITLE
test13 py file github

### PR DESCRIPTION
**Pull request by v-odemir**

### CODE REVIEW

**Comment by burakulusooy** [comment]
comment-test-1

File: test13.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
+
+class GameState: 
+        
+        def __init__(self):
+                self.board = [
+                        ["bR", "bN", "bB", "bQ", "bK", "bB", "bN", "bR"],
```
Comment (burakulusooy) [inline test13.py:6]: comment-review-1

Comment (burakulusooy) [reply test13.py:6]: comment-reply

File: test13.py [hunk 2]
```diff
@@ -218,4 +218,5 @@ def getChessNotation(self):
                 return self.getRankFile(self.startRow, self.startCol) + self.getRankFile(self.endRow, self.endCol)
 
         def getRankFile(self, r, c):
-                return  self.colsToFiles[c] + self.rowsToRanks[r]
```
Comment (burakulusooy) [inline test13.py]: comment-review-2

File: test13.py [hunk 3]
```diff
@@ -218,4 +218,5 @@ def getChessNotation(self):
...
\ No newline at end of file
+                return  self.colsToFiles[c] + self.rowsToRanks[r]
```
Comment (burakulusooy) [inline test13.py:221]: comment-review-3

File: test13.py (hunk 1 above)
Comment (v-odemir) [reply test13.py:6]: test comment 1

**Pull request merged**